mods/flora/underground.lua

```diff
@@ -0,0 +1,472 @@
+	wield_image = "flora_firevine.png",
+	paramtype = "light",
+	light_source = 8,
+	floodable = true,
+	sunlight_propagates = true,
+	walkable = false,
+	selection_box = {
+		type = "fixed",
+		fixed = {-6 / 16, -0.5, -6 / 16, 6 / 16, 0.5, 6 / 16},
+	},
+	drop = {
+		items = {
+			{
+				rarity = 1,
+				items = {"blocks:firebranches"},
+			},
+			{
+				rarity = 2,
+				items = {"blocks:firebranches"},
+			},
+		},
+	},
+	groups = {snappy = 3, flammable = 2},
+	sounds = default.node_sound_leaves_defaults(),
+	after_dig_node = function(pos, node, metadata, digger)
+		local nn = node.name
+		ccore.dig_dir(pos, {nn}, -1, digger)
+	end
+})
+
+-- Lapis (Hot sub-biome flora)
+minetest.register_node("flora:azure_moss", {
+	description = "Azure Moss",
+	tiles = {"flora_azure_moss.png"},
+	inventory_image = "flora_azure_moss.png",
+	drawtype = "firelike",
+	paramtype = "light",
+	sunlight_propagates = true,
+	walkable = false,
+	floodable = true,
+    light_source = 4,
+	drop = {
+		items = {
+			{
+				rarity = 1,
+				items = {"blocks:azure_mycelium"},
+			},
+			{
+				rarity = 3,
+				items = {"blocks:azure_mycelium"},
+			},
+		},
+	},
+	groups = {snappy = 3},
+	sounds = default.node_sound_leaves_defaults(),
+	selection_box = {
+		type = "fixed",
+		fixed = {-8 / 16, 4/16, -8 / 16, 8 / 16, 8 / 16, 8 / 16},
+	}
+})
+for i=1,5 do
+	minetest.register_node("flora:azure_grass_" .. i, {
+		description = "Azure Grass",
+		tiles = {"flora_azure_grass.png^[sheet:5x1:" .. i - 1 .. ",0"},
+		inventory_image = "flora_azure_grass.png^[sheet:5x1:" .. i - 1 .. ",0",
+		drawtype = "plantlike",
+		paramtype = "light",
+		sunlight_propagates = true,
+		walkable = false,
+		floodable = true,
+		drop = {
+			items = {
+				{
+					rarity = 1,
+					items = {"blocks:azure_mycelium"},
+				},
+				{
+					rarity = 2,
+					items = {"blocks:azure_mycelium"},
+				},
+			},
+		},
+		groups = {grass = 1, snappy = 3},
+		sounds = default.node_sound_leaves_defaults(),
+		selection_box = {
+			type = "fixed",
+			fixed = {-8 / 16, -0.5, -8 / 16, 8 / 16, -4 / 16, 8 / 16},
+		}
+	})
+	minetest.register_node("flora:azureshroom_" .. i, {
+		description = "Azureshroom\n\"Kinda hard to pronounce\"",
+		tiles = {"flora_azureshroom.png^[sheet:5x1:" .. i - 1 .. ",0"},
+		inventory_image = "flora_azureshroom.png^[sheet:5x1:" .. i - 1 .. ",0",
+		drawtype = "plantlike",
+		paramtype = "light",
+		sunlight_propagates = true,
+		walkable = false,
+		floodable = true,
+		light_source = 8,
+		drop = {
+			items = {
+				{
+					rarity = 1,
+					items = {"blocks:azure_mycelium 2"},
+				},
+				{
+					rarity = 3,
+					items = {"blocks:azure_mycelium 4"},
+				},
+			},
+		},
+		groups = {mushroom = 1, snappy = 3},
+		sounds = default.node_sound_leaves_defaults(),
+		selection_box = {
+			type = "fixed",
+			fixed = {-8 / 16, -0.5, -8 / 16, 8 / 16, -4 / 16, 8 / 16},
+		}
+	})
+end
+minetest.register_node("flora:azure_plant", {
+	description = "Azure Plant",
```

Comment:
Azure Weed? Shrub?
Naming a plant as just "plant" just feels a bit odd to me.

Reply:
I'll think of something